src/fraction.ts

```diff
@@ -134,9 +150,10 @@ export class Fraction {
     const u = this.numerator * a - otherNumerator * b;
     return this.set(u, lcm);
   }
-  multiply(param1, param2) {
-    let otherNumerator;
-    let otherDenominator;
+
+  multiply(param1?: Fraction | number, param2?: number): this {
```

Comment:
Can these be given default values of 1 and 1 also?  I feel it just helps document what happens if they're omitted.

Reply:
I agree it also simplifies the code.